modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlCommandHandler.java

```diff
@@ -338,6 +339,10 @@ protected static BiFunction<Object, Throwable, Boolean> handleModificationResult
                 }
             }
 
+            if (err instanceof CatalogValidationException) {
```

Comment:
may be better place for the convertation at SqlExceptionMapperProvider

Reply:
moved to SqlExceptionMapperProvider